README.md

```diff
@@ -57,8 +57,51 @@ If you are running from the ISO RancherOS will be running from memory.  In order
 
 ## Installing to Disk/Upgrading
 
-Coming soon (but you can guess it's all based on Docker)
+Beta:
 
+To install RancherOS on a new disk you can now use the `rancheros-install` command. 
+
+For non-ec2 installs, before getting started create a cloud-init file that will provide your initial ssh keys. At a minimum something like:
```

Comment:
Is the cloud config only used with the ec2 install?  I'll probably figure this out in a second when I read over the os-installer image...